scripts/ci-cron.sh

```diff
@@ -1,8 +1,14 @@
 #!/usr/bin/env bash
 
+set -o nounset
 set -o errexit
 
-for package in $(cat sources.txt) doc/tutorial ; do
+export PATH=$(stack path --bin-path):$PATH
+
+stack install cabal-install
+cabal update
```

Comment:
Why install `cabal-install` via `stack`? Why even install `stack` in the first place? So that we can install ghc?